lib/exercism/user_progression.rb

```diff
@@ -2,15 +2,36 @@
 class UserProgression
 
   def self.user_progress(user)
-    track_complete_exercises = Hash.new 0
-    user.exercises.each{|exe| track_complete_exercises[exe.problem.language] += 1}
+    track_complete_exercises = {}
+    user.exercises.each do |exe|
```

Comment:
what do you think about each_with_object, could work well here
ref: http://ruby-doc.org/core-2.3.0/Enumerable.html#method-i-each_with_object